src/main/java/ui/BoardAutoCreator.java

```diff
@@ -133,7 +131,7 @@ private void createWorkAllocationBoard() {
                         "It is saved under the name \"" + boardName + "\".");
     }
 
-    private void createSampleBoard() {
+    public void createSampleBoard(boolean isCalledOnStartup) {
```

Comment:
Needs a javadoc comment. Also, the parameter can be named after what it does instead of how it's used. suppressDialog?